app/build.gradle

```diff
@@ -1,4 +1,7 @@
 apply plugin: 'com.android.application'
+apply plugin: "jacoco"
```

Comment:
Use single quotes here since we're using single elsewhere and we don't really need to use double quotes anyway. To understand the difference, you can check out https://stackoverflow.com/questions/15171049/gradle-single-vs-double-quotes